src/test/resources/gradle/cucumber/simplePost.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Simple POST
+  The server should respond to simple POST requests
+
+  Scenario: Simple POST request
+    Given The server is running on port "33333"
+    When I send method "POST" for "/simple_post" with body "hello world" to host at the specified port 1 time
```

Comment:
The `1 time` thing feels like overkill tot me.

Reply:
See above!